portal/models/app_text.py

```diff
@@ -94,7 +94,7 @@ def name_key(**kwargs):
         return "{} organization consent URL".format(organization.name)
 
     @staticmethod
-    def permanent_url(generic_url, version):
+    def permanent_url(generic_url, version, languageId='en_US'):
```

Comment:
I would think passing language into every app_text call is a little cumbersome.  Would it not make more sense for app_text to be clever enough to determine this for itself?